src/commands/admin/config.ts

```diff
@@ -61,6 +61,16 @@ export default class ConfigCommand extends BotCommand {
 			);
 			return;
 		}
+		const roleFmt = async (rs: Array<string>) =>
```

Comment:
Can you make this a function on the class rather than declared every time the command runs